README.md

```diff
@@ -213,6 +213,7 @@ Any contribution is welcome!
 * [qLib](https://github.com/qLab/qLib) - A procedural asset library for SideFX Houdini.
 * [SideFX Labs](https://github.com/sideeffects/SideFXLabs) - A completely free, open-source toolset geared towards assisting Houdini users with a variety of tasks commonly used for digital content creation.
 * [xComp](https://github.com/gugenstudio/xComp) - Image Render comparing.
+* [trigger](https://github.com/masqu3rad3/trigger) - Operational code driven rigging tool for Autodesk Maya.
```

Comment:
Can  you move it up a line? We try to keep stuff alphabetical and T comes before X.

Reply:
Sure thing. Made another commit.